docs/applications/ansible/getting-started-with-ansible.md

```diff
@@ -0,0 +1,357 @@
+
+### Running Playbooks
+
+Executing a playbook is even easier than running ad-hoc commands like we did earlier! Assuming you are in the same directory as a playbook file, you run the following command:
+
+	ansible-playbook myplaybook.yml
+
+Done! If you want to see what hosts this playbook will affect without having to open up the YAML file, you can run:
+
+	ansible-playbook myplaybook.yml --list-hosts
+
+### Types of Tasks You Can Run
+
+Ansible ships with a large collection of modules that you can run as tasks or via ad-hoc commands. To see a listing of all available modules, run:
+
+	ansible-docs -l
+
+A few common core modules you might be interested in learning off the bat include:
+
+* [command - Executes a command on a remote node](http://docs.ansible.com/ansible/command_module.html)
+* [script - Runs a local script on a remote node after transferring it](http://docs.ansible.com/ansible/script_module.html)
+* [shell - Execute commands in nodes](http://docs.ansible.com/ansible/shell_module.html)
+* [mysql_db - Add or remove MySQL databases from a remote host](http://docs.ansible.com/ansible/mysql_db_module.html)
+* [mysql_user - Adds or removes a user from a MySQL database](http://docs.ansible.com/ansible/mysql_user_module.html)
+* [postgresql_db - Add or remove PostgreSQL databases from a remote host](http://docs.ansible.com/ansible/postgresql_db_module.html)
+* [postgresql_user - Adds or removes a users (roles) from a PostgreSQL database](http://docs.ansible.com/ansible/postgresql_user_module.html)
+* [fetch - Fetches a file from remote nodes](http://docs.ansible.com/ansible/fetch_module.html)
+* [template - Templates a file out to a remote server](http://docs.ansible.com/ansible/template_module.html)
+* [yum - Manages packages with the yum package manager](http://docs.ansible.com/ansible/yum_module.html)
+* [apt - Manages apt-packages](http://docs.ansible.com/ansible/apt_module.html)
+* [git - Deploy software (or files) from git checkouts](http://docs.ansible.com/ansible/git_module.html)
+* [service - Manage services](http://docs.ansible.com/ansible/service_module.html)
+
+
+## Basic Web Server Setup via Playbooks
+
+Let's get down to the business of turning a freshly created Linode server into a web server, configured with Apache, MySQL, and PHP, ready to serve up dynamic sites and configured with the proper users and permissions. For brevity we won't handle all of the features and configuration that might normally be involved, but will cover enough to get you started.
+
+This guide will assume a brand new Ubuntu 14.04 LTS server, without any additional configuration already done to the box. The very first order of business will be to add in our public encryption keys so that we can connect without supplying passwords.
+
+{: .note}
```

Comment:
I would consider removing this note entirely. Since Ansible will give you the same prompt as SSH there' no difference, and I think we can assume that someone setting up Ansible is already familiar with the RSA fingerprint warning.